apps/vs-code-designer/src/app/commands/createNewCodeProject/createCodeProjectSteps/createLogicApp/ScriptProjectCreateStep.ts

```diff
@@ -61,7 +61,7 @@ export class ScriptProjectCreateStep extends ProjectCodeCreateStepBase {
           AzureWebJobsStorage: 'UseDevelopmentStorage=true',
           WORKFLOWS_SUBSCRIPTION_ID: '',
           FUNCTIONS_WORKER_RUNTIME: 'node',
-          APP_KIND: 'workflowapp',
+          APP_KIND: logicAppKind,
```

Comment:
What is this setting? Did we rename this setting to that? I know we had workflowApp originally so I wanted to double check